build.gradle

```diff
@@ -2355,7 +2366,7 @@ project(':streams:streams-scala') {
     spotless {
       scala {
         target '**/*.scala'
-        scalafmt("$versions.scalafmt").configFile('../../checkstyle/.scalafmt.conf').scalaMajorVersion(versions.baseScala)
+        scalafmt("$versions.scalafmt").configFile('../../checkstyle/.scalafmt.conf').scalaMajorVersion('2.13')
```

Comment:
What's the motivation for this change? This means that we would run scalafmt with 2.13 rules regardless of the Scala version we compile. I think we could use something like https://scalameta.org/scalafmt/docs/configuration.html#scala-2-with--xsource3.